src/Datastore/DatastoreSessionHandler.php

```diff
@@ -130,21 +130,40 @@ class DatastoreSessionHandler implements SessionHandlerInterface
     /* @var Transaction */
     private $transaction;
 
+    /* @var array */
+    private $options;
+
     /**
      * Create a custom session handler backed by Cloud Datastore.
      *
      * @param DatastoreClient $datastore Datastore client.
      * @param int $gcLimit [optional] A number of entities to delete in the
      *        garbage collection.  Defaults to 0 which means it does nothing.
      *        The value larger than 1000 will be cut down to 1000.
+     * @param array $options [optional]  {
+     *     Configuration Options
+     *
+     *     @type array $entityOptions Default options to be passed to the
+     *           {@see \Google\Cloud\Datastore\DatastoreClient::entity()} method when writing session data to Datastore.
+     *           If not specified, defaults to ['excludeFromIndexes' => ['data']].
+     * }
      */
     public function __construct(
         DatastoreClient $datastore,
-        $gcLimit = self::DEFAULT_GC_LIMIT
+        $gcLimit = self::DEFAULT_GC_LIMIT,
+        array $options = []
     ) {
         $this->datastore = $datastore;
         // Cut down to 1000
         $this->gcLimit = min($gcLimit, 1000);
+
+        if (!isset($options['entityOptions'])) {
```

Comment:
Perhaps this should also check that the value is an array to prevent a failure on line 246.